metadata-ingestion/docs/transformer/dataset_transformer.md

```diff
@@ -21,9 +21,11 @@ The below table shows transformer which can transform aspects of entity [Dataset
 | Field                       | Required | Type    | Default       | Description                                 |
 |-----------------------------|----------|---------|---------------|---------------------------------------------|
 | `semantics`                 |          | enum    | `OVERWRITE`   | Whether to OVERWRITE or PATCH the entity present on DataHub GMS. |
```

Comment:
is this field respected? if yes, what does it do?

imo it seems like we should not inherit from `TransformerSemanticsConfigModel`